packages/core/src/Page/index.js

```diff
@@ -482,6 +486,46 @@ class Page {
     this.collectHeadingsAndKeywords(pageContent);
   }
 
+  /**
+   * Compiles page into Vue Application to get the page render function and places
+   * it into a script so that the browser can retrieve the page render function to
+   * render the page.
+   *
+   * This is to avoid the overhead of compiling the page into Vue application
+   * on the client's browser (alleviates FOUC).
+   *
+   * By pre-compiling the page, we can get the Vue render function for the page
```

Comment:
is this a leftover from the earlier one-huge-file? seems like a repeat of the first two sentences.